app-client/src/main/resources/application-docker.properties

```diff
@@ -0,0 +1,12 @@
+#SERVER CONFIGURATIONS
+server.port= 8081
+server.servlet.context-path= /cdc
+
+#SPRING SSE TIMEOUT
+spring.mvc.async.request-timeout= -1
```

Comment:
This line can be removed, once you take out `\notification` endpoint and `AppService` class

Reply:
Resolved